### PR TITLE
BugFix-0788738

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
0788738 | Unable to install SP 'Lost or Stolen Device Response', getting time out error
   > Modified playbook and Scenario name (Removed Spacial cha /)

### CODE REVIEW

File: info.json [hunk 1]
```diff
@@ -16,26 +16,22 @@
...
         }
     ],
-    "category": [],
-    "certified": "true",
-    "prerequisite": [
-        "Configure the Active Directory Connector",
-        "Configure the ServiceNow Connector",
-        "Configure the FortiEDR Connector"
-    ],
+    "prerequisite": "- Configure the Active Directory Connector\n- Configure the ServiceNow Connector\n- Configure the FortiEDR Connector",
```
Comment (cs-suyog-jinturkar) [inline info.json:23]: It should be array, right?

File: connectors/data.json [hunk 2]
```diff
@@ -1,72 +1,4 @@
...
-            "get_file": [],
-            "get_event_file": [],
-            "remediate_device": [],
-            "get_collector_list": [],
-            "isolate_collector": [],
-            "unisolate_collector": [],
-            "create_exception": [],
-            "list_exception": [],
-            "update_exception": [],
-            "get_event_exceptions": [],
```
Comment (cs-suyog-jinturkar) [inline connectors/data.json:55]: Why this file is removed?

File: info.json [hunk 3]
```diff
@@ -1,9 +1,9 @@
 {
-    "name": "lostOrStolenDeviceResponse",
+    "name": "lost-StolenDeviceResponse",
     "version": "1.0.0",
     "type": "solutionpack",
     "local": true,
-    "label": "Lost or Stolen Device Response",
```
Comment (cs-suyog-jinturkar) [inline info.json:6]: Why we changed the name?

File: info.json [hunk 4]
```diff
@@ -53,20 +49,11 @@
...
-                "name": "04 - Use Case - Lost _ Stolen Device Response"
+                "name": "04 - Use Case - Lost or Stolen Device Response"
             }
         ],
+        "globalVariables": [],
         "connectors": [
-            {
-                "name": "Active Directory",
-                "apiName": "activedirectory",
-                "version": "2.2.0"
```
Comment (cs-suyog-jinturkar) [inline info.json:63]: Why it was removed now?

File: info.json (hunk 1 above)
Comment (cs-sujeet-ahirrao) [reply info.json:23]: Yes it was .. now it is coming in string

File: connectors/data.json (hunk 2 above)
Comment (cs-sujeet-ahirrao) [reply connectors/data.json:55]: This connector is coming from SOAR Framework and this SP is dependent on that hence removed

File: info.json (hunk 3 above)
Comment (cs-sujeet-ahirrao) [reply info.json:6]: previously spacial char was not supported in SP name .. now they have provided the spacial char hence renamed

File: info.json (hunk 4 above)
Comment (cs-sujeet-ahirrao) [reply info.json:63]: same as above

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**